drone-localization/core/services/utils_for_refinement/Head/groupfusion.py

```
@@ -0,0 +1,172 @@
+                 single_output_channel=8,
+                 merged_linear_layer_num=1,
+                 merged_linear_channel=128,
+                 **kwargs):
+        super(MultiEnhanceGroupFusionHead, self).__init__()
+        self.out_scale = out_scale
+        self.gc_mid_channel = gc_mid_channel
+        self.pool = ChannelPool(mode=pool, linear_input_dim=muti_level_nums)
+        self.merged_linear = nn.Conv2d(muti_level_nums*single_output_channel, merged_linear_channel, kernel_size=1, stride=1, padding=0)
+
+    def forward(self, z, x):
+        # assert len(z)==len(x), "输入尺度不对应！！！"
```

Ruff (0.15.12)

[warning] Comment contains ambiguous `！` (FULLWIDTH EXCLAMATION MARK). Did you mean `!` (EXCLAMATION MARK)?

(RUF003)